common/app/experiments/Experiments.scala

```diff
@@ -0,0 +1,72 @@
+package experiments
+
+import conf.switches.Owner
+import experiments.ParticipationGroups._
+import org.joda.time.LocalDate
+import play.api.mvc.RequestHeader
+
+object ActiveExperiments extends ExperimentsDefinition {
+  val allExperiments: Set[Experiment] = Set(
+    CommercialClientLogging,
+    CommercialPaidContentTemplate,
+    ABNewDesktopHeader,
+    ABJavascriptRendering,
+    Garnett
+  )
+  implicit val canCheckAB = new CanCheckExperiment(this)
+}
+
+object CommercialClientLogging extends Experiment(
+  name = "commercial-client-logging",
+  description = "A slice of the audience who will post their commercial js performance data",
+  owners = Seq(Owner.withGithub("rich-nguyen")),
+  sellByDate = new LocalDate(2018, 2, 1),
+  participationGroup = Perc1A
```

Comment:
Ah, the [good old days](https://github.com/guardian/frontend/pull/8886). I think this is approach is much less error prone. I wonder if you should allow participation groups to be a `List` of groups, to allow the most flexibility to build the right audience size?

Reply:
The thing is this implementation is based on the assumption a request is bucketed into only 2 groups in Fastly VCL (`variant` and `control`). https://github.com/guardian/fastly-edge-cache/pull/804/files
Making the participation group dynamic would require the VCL to be aware of it which is impossible without modifying the VCL when creating the experiment (which is the thing this PR is absoluterly trying to prevent)

Reply:
So how does multi-variant work, say an A-B-C test? book out two groups?

Reply:
It doesn't support multi-variant. None of current experiments requires it and I can't remember last time we needed it.
You can simulate a 3 way multi-variant test with 2 groups assigned to participation groups of the same size. The drawback woule be that cache is partitioned 4 times rather than 3 and one bucket is "wasted". However it happens so rarely that I think we can live with it.